apps/myaccount/package.json

```diff
@@ -73,7 +73,8 @@
         "regenerator-runtime": "^0.13.9",
         "semantic-ui-react": "^2.1.3",
         "swr": "^2.0.0",
-        "uuid": "^8.3.0"
+        "uuid": "^8.3.0",
+        "@wso2is/admin.userstores.v1": "^2.26.23"
```

Comment:
When adding a dependency use the command `pnpm add @wso2is/admin.applications.v1`. And then adjust the version properly.
This will keep the dependency list sorted, improving the readability.

Reply:
Ack. This dependency is removed.